Adyen/Utilities/Throttler.swift

```diff
@@ -0,0 +1,42 @@
+//
+// Copyright (c) 2020 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import Foundation
+
+// Throttles requests.
+public final class Throttler {
```

Comment:
why do we need to throttle bin lookup calls?

Reply:
you type 16 digits + make mistakes and corrections. Every change is a network call.
Throttling will save you some network data.

Reply:
Alternatively, we can move it into CardNumberItem 🤔

Reply:
hmm 🤔 not sure if its worth the complexity TBH, it will save the shopper maximum 100KB of data? is this worth a delay 0.5 between calls and the complexity? this is not a rhetorical question 😊.

Reply:
ok, I guess it make sense :-)